### ISSUE TITLE
[Bug|Crash]: 

### ISSUE THREAD

**Issue by RingingAsp98** [Yesssssman/epicfightmod#1410]
### Have you checked if a similar issue is already reported by someone else?

- [X] I checked there are no similar issues have been reported.

### Have you read the support policy?

- [X] I read it and I accept the policy.

### Are you using the latest Epic Fight and recommended Forge version?

- [X] I checked I'm using latest Epic Fight and recommended Forge version.

### Is this issue related to mod incompatibility?

- [ ] This is a mod compatibility issue and I'm aware of the problem.

### Minecraft Version

1.16.5

### What happened?

sometimes, when I join the world the battlemode starts not working, I can't attack, use special attack, block or dodge.
it fixes only after i die.
I have a video, but it's too heavy.
tell me what you need to see what's the problem.

**Comment by fzrdwa123456** [comment]
Maybe this problem should be checked to see if the buttons conflict, or restart the game to try

**Comment by fzrdwa123456** [comment]
They probably won't fix the issue you reported because 1.16.5 is no longer supported

**Comment by RingingAsp98** [comment]
unluckily I don't think it's a button conflict...

**Comment by RingingAsp98** [comment]
I just need to know what is the problem, and maybe I can fix it by myself

**Comment by MetalKnight56** [comment]
Closing issue as 1.16.5 is no longer supported.
